types/externs.d.ts

```diff
@@ -50,6 +50,11 @@ declare global {
     [P in K]: P;
   };
 
+  /** Make all properties in T nullable. */
+  type Nullable<T> = {
+    [K in keyof T]: T[K] | null;
```

Comment:
wait @connorjclark it does go at least 1 level down, so I'm confused why your example is failing...

Reply:
> it does go at least 1 level down

yeah, now that you say this, it's not the best name. I was thinking in terms of parts of speech, trying to be parallel to `Partial`, but partial is in reference to the presence (or not) of all the object's properties, while "nullable" is commonly in reference to the immediate reference being sometimes `null`. Open to new names for "all properties can be `|null`" :)

Reply:
`NullableChildren`
`NullablePartial`
`NullableProperties`

any of those sound interesting? :)

Reply:
> any of those sound interesting? :)

yes, thanks, but not needed anymore :)